src/actions/webhook.js

```diff
@@ -53,7 +54,7 @@ var addWebhookService = function addWebhook(client, args, webhook) {
 
 	this.populateBody()
 
-	var hash = murmur.hash128(JSON.stringify(this.query)).hex()
+	var hash = murmur.x86.hash128(stringify(this.query));
```

Comment:
This murmur implementation seems x86 architecture specific. Any particular reason to use this particular lib?